### ISSUE TITLE
Swap prints with logging system

### ISSUE THREAD

**Issue by andreped** [andreped/super-ml-pets#13]
This is convenient to choose which verbose to have, when training and deploying models.

See [this example](https://github.com/andreped/livermask/blob/a5e1529d98db96044bddeab4aba9dd42672fd56c/livermask/utils/utils.py#L30) for how it can be done.

**Comment by GoldExplosion** [comment]
Can I take this task?

**Comment by andreped** [comment]
> Can I take this task?

Sure! :) I'm working on fixing the training loop atm.

**Comment by GoldExplosion** [comment]
Due to a personal emergency, I wouldn't be contributing for 3-4 days.

**Comment by andreped** [comment]
> Due to a personal emergency, I wouldn't be contributing for 3-4 days.

Oh, no worries! Hope you are doing okay. I will leave this task for you open until you come back.

**Comment by GoldExplosion** [comment]
Hi i am back, can I have this issue?

**Comment by andreped** [comment]
> Hi i am back, can I have this issue?

Of course! I have left this task for you. It is a nice task for getting to know the code base as well :)

**Comment by GoldExplosion** [comment]
Please check if this is what you had in mind https://github.com/andreped/super-ml-pets/pull/38/commits/bcfffa7586ed0f50b1c688483dfe80f953840bac

**Comment by andreped** [comment]
> Please check if this is what you had in mind [bcfffa7](https://github.com/andreped/super-ml-pets/commit/bcfffa7586ed0f50b1c688483dfe80f953840bac)

Looks that way, but please, make a PR and then I can make a proper review. However, it is probably fine.

**Comment by GoldExplosion** [comment]
sample output

```

INFO deploy_agent.py 108 CV SYSTEM [self.run]: Detect the Pets and Food in the Shop Slots
INFO deploy_agent.py 110 CV SYSTEM [self.run]: Calls [image_detection.find_the_animals]
INFO deploy_agent.py 115 CV SYSTEM [self.run]: The detected Pets and Food in the Shop is : ['cricket', 'honey']
INFO deploy_agent.py 116 GAME ENGINE [self.run]: Set Environment Shop = detected Pets and Food
INFO deploy_agent.py 125 GAME ENGINE [self.run]: Get the best action to make for the given state from the loaded model
INFO deploy_agent.py 133 GAME ENGINE [self.run]: Current Team and Shop 
<bound method storeaction.<locals>.store_action of --------------
CURRENT INFO: 
--------------
PACK:  StandardPack
TURN:  1
LIVES: 10
WINS:  0
GOLD:  0

CURRENT TEAM: 
--------------
0: < Slot pet-cricket 1-2 none 1-0 > 
    1: < Slot pet-mosquito 2-2 none 1-0 > 
    2: < Slot pet-cricket 1-2 none 1-0 > 
    3: < Slot EMPTY > 
    4: < Slot EMPTY > 
    
CURRENT SHOP: 
--------------
0: < ShopSlot-pet not-frozen 3-gold pet-cricket 1-2 none 1-0 > 
    1: < ShopSlot-food not-frozen 3-gold food-honey 0-0 status-honey-bee > 
    >
INFO deploy_agent.py 135 GAME ENGINE [self.run]: Best Action = 0 end_turn
INFO deploy_agent.py 137 GAME ENGINE [self.run]: Instruction given by the model = ()
INFO deploy_agent.py 145 GAME ENGINE [self.run]: Action is valid
INFO deploy_agent.py 174 GAME ENGINE [self.run]: Calls end_turn with parameters ()
INFO actions.py 250 MOUSE ACTION [self.end_turn]: Ends the Round
INFO actions.py 43 MOUSE ACTION [self._click]: Clicks the position end_turn
INFO deploy_agent.py 177 GAME ENGINE [self.run]: Implements the action in the Environment
```

**Comment by andreped** [comment]
Fixed in 48818737b1b0bed98d7c3f194ea78afb623f8e54
